admin/app/controllers/authenticated/certifications.js

```diff
@@ -1,13 +1,18 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
 
 export default class CertificationsController extends Controller {
   @service router;
 
+  @tracked inputId;
+
   @action
-  loadCertification() {
-    const certifId = this.inputId;
-    this.router.transitionTo('authenticated.certifications.certification', certifId);
+  loadCertification(event) {
+    event.preventDefault();
+    const certifId = this.inputId && this.inputId.trim();
```

Comment:
En utilisant lodash on peut éviter de vérifier la présence de la propriété nous même : 

```suggestion
    const certifId = _.trim(this.inputId);
```